Core/Inc/main.h

```diff
@@ -38,7 +37,7 @@ extern "C" {
 #include "stdio.h"
 #include "string.h"
 //#include "API.h"
-#include "logic_layer.h"
+//#include "logic_layer.h"

```

Comment:
instead of commenting out can you not delete this line?

Reply:
deleted